app/views/shared/_commit_message_replace_container.html.haml

```diff
@@ -0,0 +1,14 @@
+.form-group.commit_message-group
+  = label_tag 'commit_message_replace', class: 'control-label' do
```

Comment:
@liyakun No I mean just make the first argument a symbol.

`= label_tag :commit_message_replace, class: 'control-label' do`